external/src/main/java/com/redhat/cloud/policies/engine/handlers/TriggersHandler.java

```diff
@@ -691,7 +704,11 @@ public void findTriggers(RoutingContext routing) {
                 }, res -> ResponseUtil.result(routing, res));
     }
 
-    @DocPath(method = GET,
+    private Span getServerSpan(RoutingContext routing) {
+        return routing.get("io.opentracing.contrib.vertx.ext.web.TracingHandler.severSpan");
```

Comment:
sever?

Reply:
Also, shouldn't the serverSpanContext come from ``TracingHandler.serverSpanContext`` ?

Reply:
`sever` is unfortunately correct

Reply:
We obtain the server span here, not the context.